include/exadg/acoustic_conservation_equations/postprocessor/sound_energy_calculator.h

```diff
@@ -222,7 +222,11 @@ class SoundEnergyCalculator
       {
         vector u = velocity.get_value(q);
```

Comment:
For clarity, variable name should be `rho_u` in my opinion

Reply:
[caabe6f](https://github.com/exadg/exadg/pull/631/commits/caabe6f046a1795b4e88879d583d111aff8de496)